Samples/Directory.Build.props

```diff
@@ -7,21 +7,9 @@
     <LibrarySamplesTargetFrameworks>net6.0</LibrarySamplesTargetFrameworks>
```

Comment:
QUESTION: Should we relax the condition, so that `net6.0` becomes our fallback? Looks like the fallback is to not define a TFM.